File: Java/Aplicatii client-server/labJavaSpringRemotingUsingHibernate/Networking/src/main/java/dto/ParticipantDTO.java
package dto;

import java.io.Serializable;

public class ParticipantDTO implements Serializable {
    private String id;
    private String nume;
    private String echipa;
    private Integer capMotor;

    public ParticipantDTO(String id, String nume, String echipa, Integer capMotor) {
        this.id = id;
        this.nume = nume;
        this.echipa = echipa;
        this.capMotor = capMotor;
    }

    public String getId() {
        return id;
    }

    public String getNume() {
        return nume;
    }

    public String getEchipa() {
        return echipa;
    }

    public Integer getCapMotor() {
        return capMotor;
    }
}

File: C#/Aplicatii client-server/labClientServerCSharp/ClientServer/Networking/dto/ParticipantDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Networking.dto
{
    [Serializable]
    public class ParticipantDTO
    {
        private string id;
        private string nume;
        private string echipa;
        private int capMotor;

        public ParticipantDTO(string id, string nume, string echipa, int capMotor)
        {
            this.id = id;
            this.nume = nume;
            this.echipa = echipa;
            this.capMotor = capMotor;
        }

        public virtual string Id { get { return id; } }
        public virtual string Nume { get { return nume; } }
        public virtual string Echipa { get { return echipa; } }
        public virtual int CapMotor { get { return capMotor; } }
    }
}
